feature_layers/toggle-between-feature-request-modes/src/main/java/com/esri/samples/toggle_between_feature_request_modes/ToggleBetweenFeatureRequestModesSample.java

```diff
@@ -132,8 +132,8 @@ public void start(Stage stage) {
       // set the starting viewpoint for the map view
       mapView.setViewpoint(new Viewpoint(45.5266, -122.6219, 6000));
       // disable the vbox and show a progress indicator when the map view is drawing (e.g. when fetching caches)
-      mapView.addDrawStatusChangedListener(e -> {
-        boolean drawStatusInProgress = e.getDrawStatus() == DrawStatus.IN_PROGRESS;
+      mapView.drawStatusProperty().addListener((property, oldValue, newValue) -> {
+        boolean drawStatusInProgress = newValue == DrawStatus.IN_PROGRESS;
```

Comment:
It's annoying that we can't bind to the 'disabledProperty' of the VBox but I think we should go with:

```
      // display the progress indicator when the map view is drawing (e.g. when fetching caches)
      progressIndicator.visibleProperty().bind(mapView.drawStatusProperty().isEqualTo(DrawStatus.IN_PROGRESS));

      // disable the vbox when the map view is drawing (e.g. when fetching caches)
      mapView.drawStatusProperty().addListener((property, oldValue, newValue) -> 
        controlsVBox.setDisable(newValue == DrawStatus.IN_PROGRESS));
```

Reply:
@jenmerritt When the sample loads for the first time, the draw status changes causing the VBox to quickly alternate between enabled/disabled. We could potentially leave it as is or keep the VBox as always enabled if you think the blinking is distracting.